ui/raidboss/data/06-ew/raid/p2n.txt

```diff
@@ -0,0 +1,66 @@
+32.8 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6835:/
+43.6 "Spoken Cataract" sync / 1[56]:[^:]*:Hippokampos:(67F7|67F8|67F9|67FD):/
+58.3 "Spoken Cataract" sync / 1[56]:[^:]*:Hippokampos:(67F7|67F8|67F9|67FD):/
+72.0 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:680F:/
+77.2 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6835:/
+84.6 "Sewage Deluge" sync / 1[56]:[^:]*:Hippokampos:67F6:/
+111.0 "Tainted Flood" sync / 1[56]:[^:]*:Hippokampos:6809:/
+126.9 "Predatory Sight" sync / 1[56]:[^:]*:Hippokampos:680B:/
+137.3 "Shockwave (knockback)" sync / 1[56]:[^:]*:Hippokampos:6807:/
+148.6 "Dissociation" sync / 1[56]:[^:]*:Hippokampos:6804:/
+153.7 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6805:/
+165.8 "Dissociation Dive" sync / 1[56]:[^:]*:Hippokampos:6806:/
+172.5 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6D14:/
+178.2 "Coherence Flare" sync / 1[56]:[^:]*:Hippokampos:6800:/
+180.7 "Coherence Line" sync / 1[56]:[^:]*:Hippokampos:6802:/
+182.7 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6803:/
+194.8 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:680F:/
+
+201.0 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6835:/
+208.6 "Sewage Deluge" sync / 1[56]:[^:]*:Hippokampos:67F6:/
+227.1 "Tainted Flood" sync / 1[56]:[^:]*:Hippokampos:6809:/
+235.0 "Spoken Cataract" sync / 1[56]:[^:]*:Hippokampos:(67F7|67F8|67F9|67FD):/
+250.7 "Sewage Eruption 1" sync / 1[56]:[^:]*:Hippokampos:680D:/
+252.2 "Sewage Eruption 2" sync / 1[56]:[^:]*:Hippokampos:680D:/
+253.7 "Sewage Eruption 3" sync / 1[56]:[^:]*:Hippokampos:680D:/
+259.5 "Spoken Cataract" sync / 1[56]:[^:]*:Hippokampos:(67F7|67F8|67F9|67FD):/
+273.4 "Tainted Flood" sync / 1[56]:[^:]*:Hippokampos:6809:/
+279.8 "Predatory Sight" sync / 1[56]:[^:]*:Hippokampos:680B:/
+287.0 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:680F:/
+296.2 "Dissociation" sync / 1[56]:[^:]*:Hippokampos:6804:/
+301.3 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6805:/
+310.7 "Shockwave (knockback)" sync / 1[56]:[^:]*:Hippokampos:6807:/
+313.4 "Dissociation Dive" sync / 1[56]:[^:]*:Hippokampos:6806:/
+318.9 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6835:/
+325.3 "Dissociation" sync / 1[56]:[^:]*:Hippokampos:6804:/
+330.4 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6805:/
+342.3 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6D14:/
+342.5 "Dissociation Dive" sync / 1[56]:[^:]*:Hippokampos:6806:/
+342.5 "Sewage Eruption 1" sync / 1[56]:[^:]*:Hippokampos:680D:/
+344.0 "Sewage Eruption 2" sync / 1[56]:[^:]*:Hippokampos:680D:/
+345.5 "Sewage Eruption 3" sync / 1[56]:[^:]*:Hippokampos:680D:/
+347.9 "Coherence Flare" sync / 1[56]:[^:]*:Hippokampos:6800:/
+350.4 "Coherence Line" sync / 1[56]:[^:]*:Hippokampos:6802:/
+352.4 "--sync--" sync / 1[56]:[^:]*:Hippokampos:6803:/
+360.6 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:680F:/
+371.8 "Murky Depths" sync / 1[56]:[^:]*:Hippokampos:680F:/
+383.1 "Doubled Impact" sync / 1[56]:[^:]*:Hippokampos:680E:/
+
+392.3 "--middle--" sync / 1[56]:[^:]*:Hippokampos:6835:/ window 50,50 jump 588.3
```

Comment:
This jump looks wrong to me.  Did you run this through `python util/test_timeline.py`?

Reply:
You're right! I had it right in my file, but uploaded wrong value here.